Created A Main README

taljacob2 opened the pull request:
Created a main README file that explains:
- **How To Run The Project**
  - What are the **prerequisites** for running the application.
  - How to **download the project**.
  - How to **run the application**.

Thanks for @ShirleyFichman for a good structure of this README (#9).

Once this PR is merged, you may:
- close #9 
- close #12

Signed-off-by: Tal Jacob <taljacob2@gmail.com>

Yarboa reviewed (commented):
Please help me understand
This PR obsoletes #9, So why not closing #9?

taljacob2 commented:
> Please help me understand This PR obsoletes #9, So why not closing #9?

Yes this obsoletes #9.
I wasn't sure if I may close it by myself.
Please close #9 this time (once you approve this PR),
and in the future, I would do it by myself instead.

kasemAlem commented on README.md:
```diff
@@ -1 +1,64 @@
+> - *Linux:* on **terminal**.
+
+### Prerequisites
+
```
Add Git as prerequisite .

taljacob2 replied:
Is it a prerequisite though?
It is mentioned in the file that people who don't have Git installed may download the project manually by pressing the green "Code" button.

Yarboa replied:
> Is it a prerequisite though? It is mentioned in the file that people who don't have Git installed may download the project manually by pressing the green "Code" button.

@taljacob2 since there is no package to deploy/install the project
Only option to consume through git,

What audience will download through curl/download?
Let me suggest a prereq Note above git client:
Advised to use git to download this project.

MHO, no one will download it without git :),

taljacob2 replied:
Alright.
- I have added Git as a *prerequisite* for this project. 
- I removed the section that explains how to download the project *without* Git.

kasemAlem commented on README.md:
```diff
@@ -1 +1,64 @@
+> NOTE: Depending on your OS, the commands in this README should be run with the following shells:
+> - *Windows:* on **cmd** or **Powershell**.
+> - *Linux:* on **terminal**.
+
```
This section is not clear enough, 
if you have a different deployments on different OS then you better write it more clear, 
even more I suggest you remove it at all. 
please concentrate on finishing the important features be merged in main.

taljacob2 replied:
Okay. I removed this section.

kasemAlem commented on README.md:
````diff
@@ -1 +1,64 @@
+     Afterwards, enter the folder of the project by running:
+     ```sh
+     cd Beyond-07-team-4
+     ```
````
Check up Yariv's comment regarding this part, to the point : 
    -git clone ...
    - cd beyond-07-team-7
    - run the script

taljacob2 replied:
Alright. I modified the file to be more "to the point"  when downloading with Git.

taljacob2 commented:
@Yarboa @kasemAlem I have updated the `README.md` to fit your reviews. You may review it again.
By the way, I have updated the file with the *new repository name*.

paOmer reviewed (approved):
LGTM

rrasouli commented on README.md:
```diff
@@ -1 +1,67 @@
+    [Download Vagrant Here!](https://www.vagrantup.com/downloads)
+
+#### **ATTENTION:**
+In order for it all to work, please enable **virtualization hardware** on your computer. An explanation on how to do it can be found [here](https://www.virtualmetric.com/blog/how-to-enable-hardware-virtualization).
```
Syntax. 
For it all to work, please enable **virtualization hardware** on your computer.

kasemAlem replied:
Incorrect syntax, please replace: 
Ensure virtualization hardware is enabled in BIOS settings.

taljacob2 replied:
@rrasouli @kasemAlem changes were made in [72b2282](https://github.com/redhat-beyond/JobSeeker/pull/13/commits/72b228281fe69200fadfdf7e3b863e3116002644), please review.

rrasouli commented on README.md:
```diff
@@ -1 +1,67 @@
+#### Download The Project
+
+On your computer:
+1. Navigate to the folder you wish to place the project at.
```
Navigate the folder you wish to place the project at.

kasemAlem replied:
Please remove it, 
just the git clone command line and that's it

taljacob2 replied:
@rrasouli @kasemAlem changes were made in [72b2282](https://github.com/redhat-beyond/JobSeeker/pull/13/commits/72b228281fe69200fadfdf7e3b863e3116002644), please review.

ShirleyFichman reviewed (approved):
Great changes! LGTM

taljacob2 commented:
@Yarboa I have used `git reflog` back to 879b1fe and split the changes I have made into two commits (as I should have done in the first place).

Yarboa commented:
LGTM
